src/saga/mergeUtils.js

```diff
@@ -2,11 +2,12 @@ import { conflictType } from "../constants";
 
 
 function handleOriginUndefined(a, b, possibleConflictType, rowIndex, colIndex) {
-    if (a === undefined) {
+
+    if (a === undefined || a.length == 0) {
         return {result: b, conflicts: []};
     }
 
-    if (b === undefined) {
+    if (b === undefined || b.length == 0) {
```

Comment:
I'm not sure this works with how the Excel JavaScript API returns data. If you get a range, it returns a rectangular 2d array - so all the spaces where there aren't values in that rectangle, I believe it fills in with empty strings.

As such, you're not going to get a case where the length is 0 - but rather the number of non-empty-string elements is zero. I might write a function called "checkEmpty" that just counts non-empty string occurrences. This could be done w/ a one liner function, using .map.